src/04-modality-specific-files/06-physiological-and-other-continous-recordings.md

```diff
@@ -22,17 +22,21 @@ directory. For example:`task-movie_stim.tsv.gz`
 Physiological recordings such as cardiac and respiratory signals and other
 continuous measures (such as parameters of a film or audio stimuli) can be
 specified using two files: a gzip compressed TSV file with data (without header
-line) and a JSON for storing start time, sampling frequency, and name of the
-columns from the TSV. Please note that in contrast to other TSV files this one
-does not include a header line. Instead the name of columns are specified in the
-JSON file. This is to improve compatibility with existing software (FSL PNM) as
-well as make support for other file formats possible in the future. Start time
-should be expressed in seconds in relation to the time of start of acquisition
-of the first volume in the corresponding imaging file (negative values are
-allowed). Sampling frequency should be expressed in Hz. Recordings with
-different sampling frequencies and/or starting times should be stored in
-separate files. The following naming conventions should be used for column
-names:
+line) and a JSON for storing the following metadata fields:
+
+| Field name        | Definition                                                                                                                                                   |
+| :---------------- | :----------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| SamplingFrequency | REQUIRED. Sampling frequency in Hz of all columns in the file.                                                                                               |
+| StartTime         | REQUIRED. Start time in seconds in relation to the start of acquisition of the first volume in the corresponding imaging file (negative values are allowed). |
```

Comment:
This sounds like it only applies to MRI data

Reply:
Agree- what about "Start time in seconds in relation to the first data sample in the corresponding neural dataset (negative values are allowed)"

Reply:
I'm assuming "start of acquisition of the first volume" is an attempt to precisely define the onset of the first data sample. What about: "Start time in seconds in relation to the start of acquisition of the first data sample in the corresponding neural dataset (negative values are allowed)."

Reply:
Hmm. I'm noticing that this appears in `func/`. Is that going to cause problems with MEG/EEG/iEEG? Or would this be similar to a simultaneous fMRI/EEG?

Reply:
What do you mean by 

> I'm noticing that this appears in func/

?

+1 on changing to: "Start time in seconds in relation to the start of acquisition of the first data sample in the corresponding neural dataset (negative values are allowed)." as @effigies suggests

Reply:
@sappelhoff I mean that on line 7, we specify that these files occur in the `func/` subdirectory. Should this be expanded to include other directory names?

Reply:
Ah I see, yes it could be made more general, for example with a `<modality>` placeholder? The text below would have to be adjusted so that the `<matches>` placeholder does not only refer to MRI data :+1: